packages/forms/src/utils/aria-describedByTest.tsx

```diff
@@ -1,10 +1,13 @@
+import { toKebabCase } from '@tpr/core';
+
 export const CheckDescribedByTag = (
 	getByText,
 	inputElement: HTMLElement,
 	errorText: string,
+	hintText: string,
 ) => {
 	const name = inputElement.getAttribute('name');
-	const errorName = `${name}_error`;
```

Comment:
I know you changed this as a result of the kebab case question I added on a previous review, but does the fix you've applied here have a wider impact?  I wonder if anything in the main app is looking for "_error" somewhere, as the kebab case thing you corrected was really only in one place, but the fix looks like it could affect many things?

Happy for you to close off this comment and merge if you searched for and tested any impacts though in the downstream apps.

Reply:
Ok, good point, I'll check the other apps.

Reply:
None found!